upstream-community-operators/hazelcast-enterprise/hazelcast-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    alm-examples: |-
+      [{
+       "apiVersion": "hazelcast.com/v1alpha1",
+       "kind": "Hazelcast",
+       "metadata": {
+          "name": "hz"
+       },
+       "spec": {
+          "image": {
+            "tag": "3.11.2"
+          },
+          "hazelcast": {
+            "licenseKeySecretName": "hz-license-key-secret"
+          },
+          "securityContext": {
+            "runAsUser": 65534,
+            "fsGroup": 65534
+          }
+       }
+      }]
+    categories: "In-Memory Computing Platform"
+    certified: false
+    containerImage: hazelcast/hazelcast-enterprise-operator:0.0.1
+    createdAt: 2019-03-14T09:58:52Z
+    description: Install Hazelcast Enterprise cluster.
+    support: Hazelcast, Inc
+  name: hazelcast-operator.v0.0.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Hazelcast Enterprise cluster.
+      displayName: Hazelcast Enterprise
+      group: hazelcast.com
+      kind: Hazelcast
+      name: hazelcasts.hazelcast.com
+      version: v1alpha1
+  description: |
+    Hazelcast IMDG Enterprise is the most widely used in-memory data grid with hundreds of thousands of installed clusters around the world. It offers caching solutions ensuring that data is in the right place when it’s needed for optimal performance.
+
+    ### Before Your Start
+
+    Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+
+        $ kubectl create secret generic hz-license-key-secret --from-literal=key=LICENSE-KEY-HERE
+
+    ### Configuration
+
+    For the complete configuration options, please refer to the [Hazelcast Enterprise Helm Chart](https://github.com/hazelcast/charts/tree/master/stable/hazelcast-enterprise) description.
+
+  displayName: Hazelcast Operator
+  install:
+    spec:
+      deployments:
+      - name: hazelcast-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: hazelcast-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: hazelcast-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
If the AllNamespaces installMode is supported by this operator, this means that the operator supports use of the downward API for watching all namespaces, as described in OLM's documentation: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/operatorgroups.md
For this to work, `WATCH_NAMESPACE` should be set to the fieldPath for `metadata.annotations['olm.targetNamespaces']`